plan/sort_test.go

```diff
@@ -320,3 +320,310 @@ func TestSortComplex(t *testing.T) {
 		t.Error(diff)
 	}
 }
+
+func TestSort_PlanShouldBeUpdatedCorrectly(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    blip.Plan
+		expected blip.Plan
+	}{
+		{
+			name: "1 level",
+			input: blip.Plan{
+				Levels: map[string]blip.Level{
```

Comment:
It seems existing tests cover most of these test cases already? If not, then what else are these tests covering?